packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/en.ftl

```diff
@@ -10,3 +10,11 @@ sub-change-submit = Confirm change
 sub-update-current-plan-label = Current plan
 sub-update-new-plan-label = New plan
 sub-update-total-label = New total
+sub-update-one-time-charge = One-time charge (due today)
+sub-update-prorated-total = <strong>Upgrade</strong> (Prorated to end of period)
+
+## Frequency of payment for subscription plan change
+sub-update-interval-daily = Daily
+sub-update-interval-weekly = Weekly
+sub-update-interval-monthly = Monthly
+sub-update-interval-yearly = Yearly
```

Comment:
```suggestion
sub-update-interval-yearly = Yearly

##

```
Reset the group comment

Reply:
TIL! Thank you @flodolo . I didn't know about resetting.

Reference: https://firefox-source-docs.mozilla.org/l10n/fluent/review.html#comments